app/config/Modules.scala

```diff
@@ -21,7 +21,7 @@ import com.google.inject.AbstractModule
 class Modules extends AbstractModule {
 
   override def configure(): Unit = {
-    bind(classOf[AppConfig]).to(classOf[AppConfig]).asEagerSingleton()
+    bind(classOf[AppConfig]).to(classOf[BackendAppConfig]).asEagerSingleton()
```

Comment:
The "ImplementedBy" actually does this for you. Not ganna ask you to remove it, as it doesn't hurt to have it there, but just so you know for the future.